corehq/apps/reminders/tasks.py

```diff
@@ -37,23 +39,20 @@ def get_subcases(case):
             subcases.append(CommCareCase.get(index.referenced_id))
     return subcases
 
-@task(queue=settings.CELERY_REMINDER_CASE_UPDATE_QUEUE, ignore_result=True, acks_late=True)
-def case_changed(case_id, handler_ids, retry_num=0):
+
+@task(queue=settings.CELERY_REMINDER_CASE_UPDATE_QUEUE, ignore_result=True, acks_late=True,
+      default_retry_delay=CASE_CHANGED_RETRY_INTERVAL * 60, max_retries=CASE_CHANGED_RETRY_MAX,
+      bind=True)
+def case_changed(self, domain, case_id):
```

Comment:
good question - see explanation at the bottom of this section: http://docs.celeryproject.org/en/latest/userguide/tasks.html#context

by passing bind=True to the `@task` decorator, it turns it into a bound method of the task instance so that you can access attributes/methods on it (like the retry() method here)